Dockerfile-multistage-node

```diff
@@ -23,32 +23,14 @@ RUN rm -rf static templates && \
     ls -l static/css && \
     echo "--------------------------------------"
 
-FROM hydroshare/hs_docker_base:262c2ca
+FROM hydroshare/hs_docker_base:06e98a5
 
 COPY --from=node-build /hydroshare /hydroshare
 
 # Set the locale. TODO - remove once we have a better alternative worked out
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
     locale-gen
 
-RUN pip install psycopg==3.1.*
-RUN pip install django==4.2.*
-RUN pip install djangorestframework==3.14.*
-RUN pip install django-haystack==3.2.*
-RUN pip install drf-yasg==1.21.*
-RUN pip install django-robots==6.1
-RUN pip install django-autocomplete-light==3.11.*
-RUN pip install django-widget-tweaks==1.5.*
-RUN pip install django-crispy-forms==2.1
-RUN pip install crispy-bootstrap3==2024.1
-RUN pip install Mezzanine==6.0.0
-RUN pip install hsmodels==1.0.4
-RUN pip install git+https://github.com/CUAHSI/django-tus.git@2aac2e7c0e6bac79a1cb07721947a48d9cc40ec8
-
-# intentionally keep bleach at https://github.com/mozilla/bleach/releases/tag/v5.0.1
-# due to issue with mezzanine 6.0.0
-# https://github.com/stephenmcd/mezzanine/issues/2054
-RUN pip install bleach==5.0.1
```

Comment:
There are 3 more pip install lines down below for boto3, django-storages, and smart_open[s3]. Looks like it was me who put them in an unconventional spot.

Reply:
thanks I will add them to the base image